.github/workflows/CI.yml

```diff
@@ -103,9 +103,95 @@ jobs:
         shell: bash
         run: sed -i 's/rustflags = \[/rustflags = \[\n    "-Dwarnings",/' .cargo/config.toml
 
-      - name: Test compilation
-        shell: bash
-        run: cargo hack check --workspace --each-feature --no-dev-deps
+      - name: Check files changed in api_models
+        id: changed-files-api_models
+        uses: tj-actions/changed-files@v35
+        with:
+          files: crates/api_models/*.rs  # Alternatively using: `docs/**` or `docs`
```

Comment:
Is this pattern correct?

Should it be `crates/api_models` or `crates/api_models/**/*.rs` instead?